tests/DependencyInjection/Compiler/AddDependencyCallsCompilerPassTest.php

```diff
@@ -259,6 +259,92 @@ public function testProcessGroupNameAsParameter()
         $this->assertFalse(array_key_exists('%sonata.admin.parameter.groupname%', $adminGroups));
     }
 
+    public function testFixTemplatesConfiguration()
+    {
+        $container = $this->getContainer();
+
+        if (class_exists(DefinitionDecorator::class)) {
+            $vehiculeAdmin = new DefinitionDecorator('sonata_abstract_movable_object_admin');
+            $carAdmin = new DefinitionDecorator('sonata_abstract_vehicule_admin');
+        } else {
+            $vehiculeAdmin = new ChildDefinition('sonata_abstract_movable_object_admin');
+            $carAdmin = new ChildDefinition('sonata_abstract_vehicule_admin');
+        }
+
+        $carAdmin
+            ->setClass('Sonata\AdminBundle\Tests\DependencyInjection\MockAdmin')
+            ->setArguments(['', 'Sonata\AdminBundle\Tests\DependencyInjection\Car', 'SonataAdminBundle:CRUD'])
+            ->addTag('sonata.admin', ['group' => 'sonata_report_two_group', 'manager_type' => 'orm', 'show_mosaic_button' => true]);
+        $vehiculeAdmin
```

Comment:
can you please rename it to `vehicle`?

Reply:
What?! You don't like my frenglish? :smile:
Done :+1: